framework/include/base/MooseApp.h

```diff
@@ -939,6 +1002,24 @@ class MooseApp : public ConsoleStreamInterface,
   /// Pointer to the executioner of this run (typically build by actions)
   std::shared_ptr<Executioner> _executioner;
 
+  /// Pointer to the Executor of this run
+  std::shared_ptr<Executor> _executor;
+
+  /// Pointers to all of the Executors for this run
+  std::map<std::string, std::shared_ptr<Executor>> _executors;
```

Comment:
Are there next level executors?

Reply:
I'm not sure I understand?

Reply:
I meant: whether or not `std::map<std::string, std::shared_ptr<Executor>> _executors` are children of `std::shared_ptr<Executor> _executor`?

Reply:
You can code executors to store subs/children however you want.  The system doesn't presuppose any structure.